package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "stream-to-mongo-db",
-  "version": "1.1.0",
+  "name": "stream-to-mongo-db-fabienjuif",
```

Comment:
Can we change this please @fabienjuif

Reply:
Haaa @AbdullahAli just in time :)

I was going to make it a fork.
I change that, as you saw I fix error handling to :)

Reply:
Thank you for the work, there are some failing tests, can you please take a look?

Reply:
I have the feeling that tests are not concurrent safe.
I am not sure how to fix that but rewriting them :/

Reply:
Bear with me, I will do a rewrite to simplify the codebase

Reply:
@AbdullahAli I fix tests in a new commit if you want to see that